more_itertools/more.py

```diff
@@ -2522,9 +2525,9 @@ def time_limited(limit_seconds, iterable):
 
 def only(iterable, default=None, too_long=None):
     """If *iterable* has only one item, return it.
-    If it has zero items, return *default*.
-    If it has more than one item, raise the exception given by *too_long*,
-    which is ``ValueError`` by default.
+    If it has zero items, return *default*, which is *None* if not specified.
```

Comment:
Could you put this docstring back the way it was? Apologies, I am picky about this.

Reply:
@bbayles : Sure - can you clarify the scope of what you would like restored?  (the part about zero items, the part about if `too_long` is not specified, or both)?

Reply:
Both, if you could please.

Reply:
Restored!